integration-tests/metrics-v4.2.19-init.patch

```diff
@@ -0,0 +1,203 @@
++                                    <skip>true</skip>
++                                </configuration>
++                            </execution>
++                            <execution>
++                                <id>error-prone-test-compile</id>
++                                <phase>test-compile</phase>
++                                <goals>
++                                    <goal>testCompile</goal>
++                                </goals>
++                                <configuration>
++                                    <failOnError>true</failOnError>
++                                    <source>${java.version}</source>
++                                    <target>${java.version}</target>
++                                    <compilerArgs combine.children="append">
++                                        <arg>-Xpkginfo:always</arg>
++                                        <arg>-XDcompilePolicy=simple</arg>
++                                        <arg>${error-prone-support.flags}</arg>
++                                    </compilerArgs>
++                                    <annotationProcessorPaths combine.children="append">
++                                        <path>
++                                            <groupId>com.google.errorprone</groupId>
++                                            <artifactId>error_prone_core</artifactId>
++                                            <version>${error-prone.version}</version>
++                                        </path>
++                                        <path>
++                                            <groupId>tech.picnic.error-prone-support</groupId>
++                                            <artifactId>error-prone-contrib</artifactId>
++                                            <version>${error-prone-support.version}</version>
++                                        </path>
++                                        <path>
++                                            <groupId>tech.picnic.error-prone-support</groupId>
++                                            <artifactId>refaster-runner</artifactId>
++                                            <version>${error-prone-support.version}</version>
++                                        </path>
++                                    </annotationProcessorPaths>
++                                </configuration>
++                            </execution>
++                        </executions>
++                    </plugin>
++                </plugins>
++            </build>
++        </profile>
+         <profile>
+             <id>release-sign-artifacts</id>
+             <activation>
+@@ -323,15 +434,7 @@
+                     <compilerArgs>
+                         <arg>-Xlint:all</arg>
+                         <arg>-XDcompilePolicy=simple</arg>
+-                        <arg>-Xplugin:ErrorProne -XepExcludedPaths:.*/target/generated-sources/.*</arg>
```

Comment:
Do we use this? Otherwise I'd say we can leave this untouched to minimize the diff.

Reply:
Same as above